src/lib/api/ErrorHandler.js

```diff
@@ -2,7 +2,8 @@
 import os from 'os';
 import config from 'config';
 import { DEFAULT_ERROR_MESSAGE } from 'constants';
-
+import Logger from '../logging';
+Logger.warn("Dev Mode: "+ config.mode.environment+ '\n' );
```

Comment:
Maybe just only log the environment if we are in a non production environment.

Reply:
Done